grid_map_demos/src/OctomapToGridmapDemo.cpp

```diff
@@ -91,21 +118,20 @@ void OctomapToGridmapDemo::convertAndPublishMap()
     *octomap, "elevation", map_, &min_bound,
     &max_bound);
   if (!res) {
-    ROS_ERROR("Failed to call convert Octomap.");
+    RCLCPP_ERROR(this->get_logger(), "Failed to call convert Octomap.");
     return;
   }
-  map_.setFrameId(srv.response.map.header.frame_id);
+  map_.setFrameId(response->map.header.frame_id);
 
   // Publish as grid map.
-  grid_map_msgs::GridMap gridMapMessage;
-  grid_map::GridMapRosConverter::toMessage(map_, gridMapMessage);
-  gridMapPublisher_.publish(gridMapMessage);
+  auto gridMapMessage = grid_map::GridMapRosConverter::toMessage(map_);
+  gridMapPublisher_->publish(std::move(gridMapMessage));
 
   // Also publish as an octomap msg for visualization
-  octomap_msgs::Octomap octomapMessage;
+  octomap_msgs::msg::Octomap octomapMessage;
   octomap_msgs::fullMapToMsg(*octomap, octomapMessage);
   octomapMessage.header.frame_id = map_.getFrameId();
-  octomapPublisher_.publish(octomapMessage);
+  octomapPublisher_->publish(octomapMessage);
```

Comment:
This would be good to have as a pointer and move to the publisher for jitter issues